jobs/settings.py

```diff
@@ -314,3 +316,8 @@
 EMAIL_PORT = env("EMAIL_PORT", default=None)
 EMAIL_HOST_USER = env("EMAIL_HOST_USER", default=None)
 EMAIL_HOST_PASSWORD = env("EMAIL_HOST_PASSWORD", default=None)
+
+# Catpcha
+RECAPTCHA_PUBLIC_KEY = env("RECAPTCHA_PUBLIC_KEY", default="")
+RECAPTCHA_PRIVATE_KEY = env("RECAPTCHA_PRIVATE_KEY", default="")
```

Comment:
The private key shouldn't have default. Then you have to add it in docker-compose.testing.yaml like the rest of secrets that are there.
After that I'll add it in the helm chart

Reply:
Done! Thank you!